sdk/client/src/lib.rs

```diff
@@ -28,6 +28,10 @@ pub use request::{JsonRpcRequest, MethodRequest};
 mod response;
 pub use response::{MethodResponse, Response};
 
+cfg_async_or_blocking! {
+    mod move_deserialize;
```

Comment:
I think you might also need a `pub use move_deserialize::Event` otherwise users of this wont be able to use the Event type.